### ISSUE TITLE
Support XIncludes with parse=text [sf#71]

### ISSUE THREAD

**Issue by tomschr** [openSUSE/daps#71]
**Reported by taroth21 on 2012-04-23 15:18 UTC**
Usage example: For automatically including a text config file in a book:

```
<appendix>
  <title>&daps; Configuration File</title>
   <screen><xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
     href="../../etc/config"
     parse="text">
        <xi:fallback>
         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
         href="/etc/daps/config"
         parse="text"/>
        </xi:fallback>
    </xi:include>
  </screen>
 </appendix>
```

From an XML point of view, this already works with DocBook, but currently, DAPS does not support it.

**Comment by tomschr** [comment]
**Updated by fsundermeyer on 2012-07-06 14:27**
- **assigned_to**: Frank Sundermeyer --> Thomas Schraitle

**Comment by tomschr** [comment]
**Commented by fsundermeyer on 2012-07-06 14:27**
This feature is a bit problematic when it comes to profiling and distributing sources:

Profiling:
The profiled files are written to a directory that not necessarily resides on the same level as the original directory, so relative paths will most probably not work after profiling.

Distributing:
When creating source archives (e.g. results from targets like locdrop or package-source), the text file includes are not part of the archive, because DAPS ATM has no means to find out which additional files need to be packed.

The following would be needed for a proper implementation:
- adding these text-file includes to filelists (requires separate handling and a new stylesheets)
- copying the test files into the profiling directory
- rewriting the links to the text files (so they point tothe copied ones) in the profiled sources

**Comment by tomschr** [comment]
**Updated by tom_schr on 2012-08-07 11:58**
- **labels**:  --> xslt

**Comment by tomschr** [comment]
**Updated by tom_schr on 2012-12-07 07:32**
- **priority**: high --> low

**Comment by tomschr** [comment]
**Updated by tom_schr on 2013-01-15 14:20**
- **status**: open --> wont-fix

**Comment by tomschr** [comment]
**Commented by tom_schr on 2013-01-15 14:20**
This feature is currently not possible with profiling and XInclude parse=text. The stylesheets cannot find the filenames. This is very unfortunate, but there is no solution at the moment.

**Comment by tomschr** [comment]
Seems we have a new idea...

**Comment by tomschr** [comment]
@taroth21 When the feature is ready, could you test it please?

**Comment by fsundermeyer** [comment]
Finally implemented with commit 22c7c3af39690011f302218b5d39079434e08bb5
